spec/requests/blocked_ip_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe BlockedIp, type: :request do
+  before( :all ) do
```

Comment:
the DB will be cleaned in between each spec, so no need to have this before all block